modules/drivers/lidar/lidar_robosense/include/parser/robosense_status.h

```diff
@@ -0,0 +1,93 @@
+/******************************************************************************
+ * Copyright 2021 The Apollo Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *****************************************************************************/
+
+#pragma once
+#include <memory>
+#include <string>
+#include <utility>
+#include <vector>
+
+#include "cyber/cyber.h"
+#include "modules/drivers/lidar/lidar_robosense/proto/sensor_suteng.pb.h"
+#include "modules/drivers/lidar/lidar_robosense/proto/sensor_suteng_conf.pb.h"
+#include "modules/drivers/proto/pointcloud.pb.h"
```

Comment:
header orders. 

pb files should be placed in its own group and before `cyber.h`, right @changsh726 ?

Reply:
other modules ’s header orders are not as you said